Adding Methylbetaine as synonym for TMAP

wkumler opened the pull request:
Added methylbetaine as a synonym for TMAP/beta-alanine-betaine, but the edit removed the quotations around everything. Here's hoping the merge goes smoothly - to quote Regina, "What's the worst that can happen?"

R-Lionheart commented:
Indeed, what is the worst that could happen... don't answer that. I'm sorry I only got to this now, I am fixing the notification settings on GitHub so I don't lose track of any pull requests. 

I checked and the mod did remove the quotes, but I'm not sure it's necessary as it can still be downloaded and works fine in all the code. What do you think?

wkumler commented:
At this point, it's probably easier to close this pull request and retry with a fresh fork - I'm pretty sure there's been some changes since the last one that look messy to merge. I'll get started on that now.
